pkg/apiserver/authentication/token/jwt.go

```diff
@@ -45,22 +46,42 @@ type jwtTokenIssuer struct {
 	maximumClockSkew time.Duration
 }
 
-func (s *jwtTokenIssuer) VerifyWithoutClaimsValidation(tokenString string) (user.Info, TokenType, error) {
+func (s *jwtTokenIssuer) VerifyWithoutClaimsValidation(tokenString string) (userInfo user.Info, tokenType TokenType, err error) {
 	parser := &jwt.Parser{
 		SkipClaimsValidation: true,
 	}
-	token, err := parser.Parse(tokenString, s.keyFunc)
-	if err != nil {
-		if token != nil {
-			mapClaims := token.Claims.(jwt.MapClaims)
 
-			return &user.DefaultInfo{
-				Name: mapClaims["sub"].(string),
-			}, "clm.TokenType", nil
+	// set up the default values
+	tokenType = "clm.TokenType"
```

Comment:
Why is default token type `clm.TokenType`?

Reply:
I just copied the code from [KubeSphere code base](https://github.com/kubesphere/kubesphere). I prefer to keep it until we have enough time to refactor it.

Reply:
Got it.